aws/ruleset.go

```diff
@@ -29,14 +33,43 @@ func (r *RuleSet) ApplyConfig(body *hclext.BodyContent) error {
 		return nil
 	}
 
+	preset := map[string]bool{}
```

Comment:
I couldn't make this work. The tests weren't passing. I can get back to this but it will take me some weeks to be able to focus on this change. I think this feature would be a nice addition to the repo!